ui/hooks/useTokensToSearch.js

```diff
@@ -119,7 +119,7 @@ export function useTokensToSearch({ usersTokens = [], topTokens = {} }) {
       others: [],
     };
 
-    memoizedTokensToSearch.forEach((token) => {
+    [...memoizedTokensToSearch, ...memoizedUsersToken].forEach((token) => {
```

Comment:
If you know about more elegant way of doing this, let me please know!

Reply:
This makes sense but from a product perspective, I'd be interested in knowing if Gal would prefer that the native currency always display first.  Just a thought worth exploring.